### PR TITLE
ADNI_to_BIDS : PET : Wrong preprocessing sequence --> Results: Impossible to make bids of FDG PET : Uniform

### PR DESCRIPTION
**Pull request by HuguesRoy**
Wrong preprocessing sequence for "STEP4_8MM" and "STEP4_6MM".

Results: Impossible to create a BIDS of PET images preprocessed step 4.

Fix issue #1157

### CODE REVIEW

File: clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py [hunk 1]
```diff
@@ -18,8 +18,8 @@ class ADNIPreprocessingStep(Enum):
     STEP1 = "Co-registered Dynamic"
     STEP2 = "Co-registered, Averaged"
     STEP3 = "Coreg, Avg, Standardized Image and Voxel Size"
-    STEP4_8MM = "Coreg, Avg, Std Img and Voxel Size, Uniform Resolution"
-    STEP4_6MM = "Coreg, Avg, Std Img and Voxel Size, Uniform 6mm Res"
+    STEP4_8MM = "Coreg, Avg, Std Img and Vox Siz, Uniform Resolution"   # noqa
```
Comment (NicolasGensollen) [inline clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py:21]: Looks like I was wrong...
You can try adding "Vox" and "Siz" here:

https://github.com/aramis-lab/clinica/blob/31543c22118914e56165c28fd525c4fe5ded9987/pyproject.toml#L113

File: clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py [hunk 2]
```diff
@@ -18,8 +18,8 @@ class ADNIPreprocessingStep(Enum):
     STEP1 = "Co-registered Dynamic"
     STEP2 = "Co-registered, Averaged"
     STEP3 = "Coreg, Avg, Standardized Image and Voxel Size"
-    STEP4_8MM = "Coreg, Avg, Std Img and Voxel Size, Uniform Resolution"
-    STEP4_6MM = "Coreg, Avg, Std Img and Voxel Size, Uniform 6mm Res"
+    STEP4_8MM = "Coreg, Avg, Std Img and Vox Siz, Uniform Resolution"  # noqa
```
Comment (NicolasGensollen) [inline clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py:21]:
```suggestion
    STEP4_8MM = "Coreg, Avg, Std Img and Vox Siz, Uniform Resolution"  # codespell:ignore
```

File: clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py [hunk 3]
```diff
@@ -18,8 +18,8 @@ class ADNIPreprocessingStep(Enum):
...
+    STEP4_6MM = "Coreg, Avg, Std Img and Vox Siz, Uniform 6mm Res"  # noqa
```
Comment (NicolasGensollen) [inline clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py:22]:
```suggestion
    STEP4_6MM = "Coreg, Avg, Std Img and Vox Siz, Uniform 6mm Res"  # codespell:ignore
```

File: pyproject.toml [hunk 4]
```diff
@@ -110,4 +110,4 @@ line-ending = "auto"
 summary = ''
 skip = ".git,LICENSE.txt,*.m"
 quiet-level = 3
-ignore-words-list = "nd,fwe,commun,fo,te,artic"
+ignore-words-list = "nd,fwe,commun,fo,te,artic,Vox,Siz"
```
Comment (NicolasGensollen) [inline pyproject.toml:113]:
```suggestion
ignore-words-list = "nd,fwe,commun,fo,te,artic"
```

**Review by NicolasGensollen — COMMENTED** [review]
Still not working... Let's try something else...

File: clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py [hunk 5]
```diff
@@ -18,8 +18,8 @@ class ADNIPreprocessingStep(Enum):
     STEP1 = "Co-registered Dynamic"
     STEP2 = "Co-registered, Averaged"
     STEP3 = "Coreg, Avg, Standardized Image and Voxel Size"
-    STEP4_8MM = "Coreg, Avg, Std Img and Voxel Size, Uniform Resolution"
-    STEP4_6MM = "Coreg, Avg, Std Img and Voxel Size, Uniform 6mm Res"
+    STEP4_8MM = "Coreg, Avg, Std Img and Vox Siz, Uniform Resolution"  # codespell:ignore
```
Comment (NicolasGensollen) [inline clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py:21]:
```suggestion
    STEP4_8MM = (
        "Coreg, Avg, Std Img and Vox Siz, Uniform Resolution"  # codespell:ignore
    )
```

**Review by NicolasGensollen — APPROVED** [review]
Alright ! I finally got it...  ⚔️ 
You need to specify the words in lower case, I completely forgot about that. 😓

**Review by AliceJoubert — APPROVED** [review]

**Pull request merged**